.github/dependabot.yml

```
@@ -12,3 +12,14 @@ updates:
     directory: /
     schedule:
       interval: monthly
+
+
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: monthly
+
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: daily

```

Lets make this less noisy
```suggestion
      interval: monthly

```